lab_2_keywords_cooccurrence/main.py

```diff
@@ -3,12 +3,79 @@
 Extract keywords based on co-occurrence frequency
 """
 from pathlib import Path
-from typing import Optional, Sequence, Mapping
+from typing import Optional, Sequence, Mapping, Any
+import re
+import json
+from math import floor
 
 KeyPhrase = tuple[str, ...]
 KeyPhrases = Sequence[KeyPhrase]
 
 
+def check_input(user_input: Any, required_type: type) -> bool:
+    """
+    Checks if the input is as required, and it is not empty (for int, float and str)
+    """
+    if user_input and isinstance(user_input, required_type):
```

Comment:
return just `user_input and isinstance(user_input, required_type)`

Reply:
Done